crate/operator/operations.py

```diff
@@ -600,6 +602,7 @@ async def suspend_or_start_cluster(
     old: kopf.Body,
     data_diff_items: kopf.Diff,
     scale_backup_metrics: bool,
+    scale_grand_central: bool,
```

Comment:
Wonder if we need this param at all? Why scale it conditionally?

Reply:
hm, the idea was to not scale it [here](https://github.com/crate/crate-operator/blob/master/crate/operator/operations.py#L1165) and [here](https://github.com/crate/crate-operator/blob/master/crate/operator/operations.py#L1215) (similar to the backup metrics deployment). But actually it might not be necessary, I'll change it..